robot/rospackages/src/mcu_control/scripts/AntennaNode.py

```diff
@@ -8,60 +8,69 @@
 from geometry_msgs.msg import Point
 #from mcu_control.msg import RoverPosition, AntennaGoal
 
+
 def subscriber_callback(message):
     rospy.loginfo(message)
     if message.x < -900 or message.y < -900:
         return
     rover['latitude'] = message.x
     rover['longitude'] = message.y
 
-    if gotAntennaPos:
-        BS_to_Rover_dir = Direction(antenna['latitude'], antenna['longitude'], rover['latitude'], rover['longitude'])
-        BS_to_Rover_dis = Distance(antenna['latitude'], antenna['longitude'], rover['latitude'], rover['longitude'])
+    if rospy.get_param('got_antenna_pos'):
```

Comment:
"got antenna piece of shit" :joy:

Reply:
I decided to change this to "got_antenna_position" LMAO